pkg/cmd/secret/set/set.go

```diff
@@ -183,6 +188,11 @@ func setRun(opts *SetOptions) error {
 	}
 
 	encoded := base64.StdEncoding.EncodeToString(eBody)
+	if opts.ShouldNotStore {
+		encodedSecret := fmt.Sprintf("Your encoded secret is %s", encoded)
```

Comment:
It's better just to print the raw value than decorating it in a sentence. You should assume that the output will be redirected to a file or a script to capture the encrypted value, and in those cases the `Your encoded secret is ` prefix will be useless and will need to be stripped away every time.